src/sonic-config-engine/tests/sample_output/py3/qos-arista7260-dualtor.json

```diff
@@ -1849,202 +1849,394 @@
         },
+        "Ethernet184|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet188|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet188|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet192|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet192|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet196|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet196|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet200|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet200|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet204|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet204|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet208|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet208|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet212|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet212|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet216|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet216|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet220|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet220|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet224|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet224|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet228|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet228|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet232|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet232|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet236|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet236|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet240|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet240|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet244|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet244|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet248|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet248|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet252|5": {
             "scheduler": "scheduler.0"
         },
+        "Ethernet252|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet0|6": {
             "scheduler": "scheduler.0"
         },
         "Ethernet4|6": {
             "scheduler": "scheduler.0"
-        },
```

Comment:
fix indent?

Reply:
Fixed. The indent change is caused by manually editing. The template is not changed.